docs/fast_data/connectors/debezium_connect_cdc.md

```diff
@@ -0,0 +1,162 @@
+---
+id: debezium_connect_cdc
+title: Debezium Connect CDC
+sidebar_label: Debezium Connect CDC
+---
+
+This page provides instructions on how to configure Debezium as a Kafka Connect connector rather than as standalone server.
+
+## Overview
+
+Unlike [Debezium Server](/fast_data/connectors/debezium_cdc.md), the configuration discussed here is not related to any plugins in the Mia-Platform Marketplace. However, you can use the official public image `quay.io/debezium/connect`.
+
+This image serves as a [Kafka Connect](https://kafka.apache.org/documentation/#connect) worker with all the Debezium connectors Debezium connectors already included.
+
+Compared to Debezium Server, Debezium Connect does not require Redis for storing metadata regarding offsets and schema management. Instead, it uses dedicated Kafka topics for this internal information.
+
+:::info
+For more details on how Debezium works we recommend checking out the [official Debezium documentation](https://debezium.io/documentation/reference/2.7/)..
+:::
+
+## Environment variables
+For a complete list of environment variables that you can configure in the container, refer to the [Official Documentation](https://github.com/debezium/container-images/tree/main/connect/2.7#environment-variables).
+
+Below is an example of `.env` file that you can import directly in the Microservices section of the console.
+
```

Comment:
@Lucascanna here I see a lot of public/private variables. can you add a small section explaining which should be private/public and their content?

Reply:
I have added a warning